src/dispatchers/mouseDispatcher.ts

```diff
@@ -183,7 +180,7 @@ module Plottable.Dispatchers {
         let newMousePosition = this._translator.computePosition(event.clientX, event.clientY);
         if (newMousePosition != null) {
           this._lastMousePosition = newMousePosition;
-          callbackSet.callCallbacks(this.lastMousePosition(), event);
+          this._callCallbacksForEvent(eventName, this._lastMousePosition, event);
```

Comment:
... strange, I don't remember changing this, but I'll put it back the way it was.